test/src/test/java/org/corfudb/runtime/view/StreamViewTest.java

```diff
@@ -36,7 +40,7 @@
 
     @Before
     public void setRuntime() throws Exception {
-        r = getDefaultRuntime().connect();
+        r = getDefaultRuntime();
```

Comment:
are you sure?

Reply:
I'm sure, getDefaultRuntime() already connects.